frontend/src/app/features/details/associates/AssociateSiteComponent.tsx

```diff
@@ -95,36 +96,19 @@ const AssociateSiteComponent: React.FC<IAssociateSiteComponent> = ({
         {viewMode === SiteDetailsMode.EditMode &&
           userType === UserType.Internal && (
             <div className="d-flex gap-2 flex-wrap ">
-              <button
-                id="add-associate-btn"
-                className=" d-flex align-items-center associate-btn"
-                type="button"
-                onClick={handleAddAssociate}
-                aria-label={'Add Associated Site'}
```

Comment:
Does the new button component set the Aria label property automatically?  I'm concerned that losing this property will impact our accessibility requirements. Please let me know if it's not an issue.

Reply:
Screen readers can infer the accessible label from the button's body and its role from the fact that it's a button. 

Generally, you only need to add `aria-label`s when there's no text for the screen reader to interpret.

```html
// This is redundant, label is automatically read
<button aria-label="Add Associated Site> Add Associated Site </button>

// No text in the body, gotta add an accessible label
<button aria-label="Add Associated Site> <PlusIcon/> </button>
```

Reply:
Right, so the onus is on us to give the buttons names with adequate text. Ex: "Add Associated Site" and not just "Add" which might be ambiguous to a screen reader user because context is not as readily apparent. That makes sense. Thanks for the explanation.